deriv_api/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+.. include:: ../README.md
+.. include:: ../docs/usage_examples.md
+"""
+
+__pdoc__ = {
+    'deriv_api.errors' : False,
+    'deriv_api.utils' : False,
+    'deriv_api.custom_future': False
+}
+
+from .deriv_api import DerivAPI
+from .errors import *
```

Comment:
```suggestion
from .deriv_api import DerivAPI
from .errors import APIError, ConstructionError, ResponseError
```
IMHO is better to state explicitly the imported classes here and use them directly from `deriv_api` module to avoid the `deriv_api.deriv_api` in imports or `from deriv_api import deriv_api` it looks redundant.

Reply:
Of course, this will require the other modules to have their imports rewritten in the form `from deriv_api import DerivAPI`.